math/fibonacci.cpp

```diff
@@ -24,8 +24,39 @@ int fibonacci(unsigned int n) {
     return fibonacci(n - 1) + fibonacci(n - 2);
 }
 
+/**
+ * Function for testing the fibonacci() function with a few
+ * test cases and assert statement.
+ */
+void test() {
```

Comment:
```suggestion
static void test() {
```
This is to specify that the function is only for this particular file and not to be globally visible.